README.md

```diff
@@ -91,4 +158,4 @@ Please submit a pull request if you know any resources that might be helpful to
 Don't forget to check out OKP4 health files:
 
 - [Contributing](https://github.com/okp4/.github/blob/main/CONTRIBUTING.md)
-- [Code of conduct](https://github.com/okp4/.github/blob/main/CODE_OF_CONDUCT.md)
+- [Code of conduct](https://github.com/okp4/.github/blob/main/CODE_OF_CONDUCT.md)
```

Comment:
`MD047/single-trailing-newline`- Files should end with a single newline character

```suggestion
- [Code of conduct](https://github.com/okp4/.github/blob/main/CODE_OF_CONDUCT.md)

```